schema support Chinese (#975)
vesoft-inc/nebula-docs: docs-2.0/3.ngql-guide/1.nGQL-overview/keywords-and-reserved-words.md

!!! Note

    - Keywords are case-insensitive.

>     - To use Chinese characters, it's required to quote them with backticks.

Reviewer: ```suggestion
    - To use Chinese characters, put them in backticks.
```

Reviewer 2: Non-English (e.g, Chinese)

    - Keywords are case-insensitive.

>     - To use Non-English (e.g, Chinese), put them in backticks.

Reviewer: ```suggestion
    - To use Non-English (e.g, Chinese) characters, put them in backticks.
```
